### PR TITLE
add qualified service in dataspace coverage API

### PR DESCRIPTION
**Pull request by YannanGao-gs**
#### What does this PR do / why is it needed ?

Improvement

add qualified service in dataspace coverage API

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by YannanGao-gs** [comment]
contains changes in https://github.com/finos/legend-engine/pull/3071

File: legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/DataSpaceAnalyticsHelper.java [hunk 1]
```diff
@@ -187,17 +187,62 @@ private static MappingModelCoverageAnalysisResult buildMappingModelCoverageAnaly
...
                 PureModelContextData classes = PureModelContextDataGenerator.generatePureModelContextDataFromClasses(coveredClasses, clientVersion, pureModel.getExecutionSupport());
                 PureModelContextData enums = PureModelContextDataGenerator.generatePureModelContextDataFromEnumerations(coveredEnumerations, clientVersion, pureModel.getExecutionSupport());
                 PureModelContextData _profiles = PureModelContextDataGenerator.generatePureModelContextDataFromProfile((RichIterable<Profile>) mappingModelCoverageAnalysisResult._profiles(), clientVersion, pureModel.getExecutionSupport());
                 PureModelContextData associations = PureModelContextDataGenerator.generatePureModelContextDataFromAssociations(mappingModelCoverageAnalysisResult._associations(), clientVersion, pureModel.getExecutionSupport());
                 mappingModelCoverageAnalysisResultProtocol.model = builder.build().combine(classes).combine(enums).combine(_profiles).combine(associations);
+                //compile
```
Comment (MauricioUyaguari) [inline legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/DataSpaceAnalyticsHelper.java:245]: lets remove this comment for now

File: legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/DataSpaceAnalyticsHelper.java [hunk 2]
```diff
@@ -187,17 +187,62 @@ private static MappingModelCoverageAnalysisResult buildMappingModelCoverageAnaly
...
+                                    String runtimePath = pureModel.getRuntimePath(keyedParam._runtime());
+                                    if (!(qualifiedMappingPath.contains(mappingPath) && qualifiedRuntimePath.contains(runtimePath)))
+                                    {
+                                        qualifiedService = false;
+                                    }
+                                }
+                                return qualifiedService;
+                            }
+                        }
+                        return false;
```
Comment (MauricioUyaguari) [inline legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/DataSpaceAnalyticsHelper.java:223]: is this expected. are we not including all non service executables ?

File: legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/DataSpaceAnalyticsHelper.java [hunk 3]
```diff
@@ -187,17 +187,62 @@ private static MappingModelCoverageAnalysisResult buildMappingModelCoverageAnaly
...
+                    }
+                    catch (Exception ignored)
+                    {
+                        return true;
```
Comment (MauricioUyaguari) [inline legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/DataSpaceAnalyticsHelper.java:227]: if the above is false, then why is this true ?

**Review by MauricioUyaguari — CHANGES_REQUESTED** [review]

**Comment by YannanGao-gs** [comment]
will be closed by https://github.com/finos/legend-engine/pull/3080

**Pull request closed without merging**